### ISSUE TITLE
[v1.1.8] publicPath does not work with protocol/absolute URLs 

### ISSUE THREAD

**Issue by antokara** [webpack-contrib/file-loader#252]
I could be misusing this option but up to/including 1.1.7,
using the **publicPath** for absolute URLs like `publicPath: "https://my.cdn/"` was possible.

In **1.1.8** due to this change:
https://github.com/webpack-contrib/file-loader/commit/26e47ca#diff-1fdf421c05c1140f6d71444ea2b27638R46
(replacing concat with path.join)
it is not possible and the **//** characters are getting replaced with one **/** rendering the absolute URL invalid...

Is there another way to configure CDN / custom absolute URLs?
ie. I want to serve some files from a different protocol/host than the rest of the app...

**Comment by aptx4869** [comment]
It also breaks webpack-dev-server
`http://localhost:8080/fontello.woff` => `http://localhost:8080/localhost:8080/fontello.woff`

**Comment by jRichardeau** [comment]
It also breaks nested relative path : 
`publicPath: "my/path"` replaced by `"my\\path"` which is not working on browser.

**Comment by WolfspiritM** [comment]
This also breaks our case using url-loader while having output.publicPath set to for example "./lib/":
```js
      {
        test: /\.(gif|png|svg|jpeg|jpg)(\?v=[a-z0-9]\.[a-z0-9]\.[a-z0-9])?$/,
        use: [
            {
                loader: 'url-loader',
                options: {
                    limit: 8192,
                    name: '[name].[ext]',
                    outputPath: 'images/',
                    publicPath: '../'
                }
            }
        ]
      }
```
Old Version produced: ```background-image: url(../images/image.jpg);```
New Version produces: ```background-image: url(../image.jpg);```
It seems to ignore the outputPath?!

**Comment by WolfspiritM** [comment]
Okay thanks.
I assumed that something like that would work but as a minor version change should not break things I expected this to be a bug rather then a fix.

**Comment by michael-ciniawsky** [comment]
Released in [`v1.1.9`](https://github.com/webpack-contrib/file-loader/releases/tag/v1.1.9) 🎉
